src/scripts/Player.ts

```diff
@@ -18,6 +18,8 @@ class Player {
     private _shinyList: boolean[];
     private _notCaughtBallSelection: GameConstants.Pokeball;
     private _alreadyCaughtBallSelection: GameConstants.Pokeball;
+    private _sortOption: KnockoutObservable<string>;
+    private _sortDirection: KnockoutObservable<string>;
```

Comment:
It is essentially just a boolean, but the select element always seems to have a string as it's value